### ISSUE TITLE
Add support for `.globalconfig`

### ISSUE THREAD

**Issue by ChristopherHaws** [OmniSharp/omnisharp-roslyn#2631]
Hey there! I have a [`.globalconfig`](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-files#global-analyzerconfig) file in the root of my projects repo but changes to it don't seem to get picked up by the LSP. In order for the changes to take effect, I have to restart VS Code.

I have tried adding the config file to `Directory.Build.props`, but the LSP still doesn't detect the changes and MSBuild complains about there being duplicate files added.
```xml
<ItemGroup Label="GlobalAnalyzers">
    <GlobalAnalyzerConfigFiles Include="$([System.IO.Path]::Combine($(MSBuildThisFileDirectory), '.globalconfig'))" />
</ItemGroup>
```

**Comment by JoeRobich** [comment]
It is likely we are not creating file watchers for global configs.
